docs/settings/apm-settings.asciidoc

```diff
@@ -13,3 +13,16 @@ You do not need to configure any settings to use APM. It is enabled by default.
 
 xpack.apm.enabled:: Set to `false` to disabled the APM plugin {kib}. Defaults to
 `true`.
+
+xpack.apm.ui.enabled:: Set to `false` to hide the APM plugin {kib} from the menu. Defaults to
+`true`.
+
+apm_oss.indexPattern:: Index pattern used to query APM data. Defaults to `apm-&#42;`.
+
+apm_oss.errorIndices:: Indicides containing error documents. Defaults to `apm-&#42;-error-&#42;`.
```

Comment:
This should still be a pattern to match against indices right? How about something like: _Index pattern to match against indices containing error documents_ ? (Same for the other event types).

Reply:
I just wanted to avoid the terribly loaded word "index pattern". But I like your sentence better, so will change.

Reply:
You might also use _Regexp to match against indices containing error documents._, then you could avoid reusing _pattern_.